static/scripts/login.js

```diff
@@ -155,14 +155,7 @@ $(document).ready(() => {
 	if ($oauthErrorLogout && $oauthSystems.length > 0 && $oauthErrorLogout.eq(0).text()) {
 		const logoutErrorOrProvider = $oauthErrorLogout.eq(0).text();
```

Comment:
this variable can be renamed to provider

Reply:
Done